lib/defaults/defaults.go

```diff
@@ -764,3 +765,11 @@ const (
 	// TeleportConfigVersionV2 is the teleport proxy configuration v2 version.
 	TeleportConfigVersionV2 string = "v2"
 )
+
+// Default values for TSH and TCTL commands.
+const (
+	DefaultSearchSessionPageLimit = "50"
```

Comment:
It's not clear to me that this is for tsh/tctl. Can you think of a better name or better place to put this constant?

Reply:
how about defaults.TshTctlSessionListLimit